packages/dropdowns/src/styled/StyledBareInput.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import StyledSelect from './StyledSelect';
+
+const CssHiddenStyling = css`
```

Comment:
Whenever this is ready for prime time, please remember to pull in the latest from https://github.com/zendeskgarden/react-components/pull/300.